static/js/simple_register_lookup_core.js

```diff
@@ -337,6 +343,20 @@ function RegistersTableRequester()
             CURRENT_APP_URL,
             data,
             function(data) {
+                for (i = 0; i < data.length; i++) {
+                  tagsToReplace = data[i][4].replace(" ","").split(",");
+                  tagsData = ""
+                  for(t = 0; t < tagsToReplace.length; t++) { 
+                     if(typeof TAG_MAPPINGS[tagsToReplace[t]] !== "undefined"){
+                        tagsData += '<a href="https://labjack.com'+ TAG_MAPPINGS[tagsToReplace[t]] +'" target="_blank">' + tagsToReplace[t]+ '</a>, '
+                        console.log(tagsToReplace[t]);
+                        console.log(TAG_MAPPINGS[tagsToReplace[t]]);
```

Comment:
We probably don't want console.logs eventually, so please either remove these now or put a comment to remove them in the future (e.g. if you need to debug in production, which would make me nervous ;-) ).